### ISSUE TITLE
Consistent Spontaneous Crash Within a Couple Minutes of Starting/Loading a Game

### ISSUE THREAD

**Issue by agsalami** [CleverRaven/Cataclysm-DDA#19898]
Just started with the latest update put out today.
I can't play a file for more than a minute or two without the game suddenly crashing. I've started new files each time and had the same result.
Tried reinstalling & restarting to eliminate corrupted files & general transient windows wonkiness as causes, with no improvement.

Generated a new world to test if something screwy happened in the initial worldgen. Same issue.

Windows 10 x64, Intel i5-6300HQ, GTX 960M GPU

**Comment by agsalami** [comment]
I've gone through and tested builds 6097, 6096, 6095, 6094 & finally 6088.
I can confirm that the same issue occurs in every version from 97-94.
When I loaded up 88 I was able to play. I have not tested 89-93, but whatever the problem is was introduced somewhere in or between 6089-6094.

I have been using the same character template in each attempt to try to limit the number of free variables, I can post the details for that character if somebody finds it relevant.
